headers/types/dungeon_mode/enums.h

```diff
@@ -1259,4 +1259,11 @@ enum gen_item_stickiness {
     GEN_ITEM_STICKY_NEVER = 2,
 };
 
+// Used to determine which version of a dungeon to load.
+enum game {
```

Comment:
Hmm I know I suggested `game` but actually I don't like how generic that is. I'm going to change it to `game_id`.

Reply:
Done.